src/vpc_scripts/groups.vgc

```diff
@@ -133,8 +133,8 @@ $Group "game"
 	"client"
 	"datacache"
 	"datamodel"
-	"dedicated"
-	"dedicated_main"
+//	"dedicated"
```

Comment:
This is not needed on Windows, please figure out another solution for this or why dedicated can't be part of the same build.

Reply:
Reverted. dedicated builds fine, but it will fail to launch due to the game failing to load `SDLMgrInterface001` in `CMaterialSystem::Connect`. The code is inside of `#if !defined(DEDICATED)`, which won't get defined unless the DEDICATED macro is defined in VPC